[2025-03-10T14:35:52.305+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-10T14:35:52.326+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xkcd_daily.dbt_run manual__2025-03-10T14:20:39.848994+00:00 [queued]>
[2025-03-10T14:35:52.332+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xkcd_daily.dbt_run manual__2025-03-10T14:20:39.848994+00:00 [queued]>
[2025-03-10T14:35:52.332+0000] {taskinstance.py:2867} INFO - Starting attempt 6 of 7
[2025-03-10T14:35:52.342+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_run> on 2025-03-10 14:20:39.848994+00:00
[2025-03-10T14:35:52.347+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=435) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-10T14:35:52.346+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'xkcd_daily', 'dbt_run', 'manual__2025-03-10T14:20:39.848994+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/XKCD_daily/dag.py', '--cfg-path', '/tmp/tmpar164ivy']
[2025-03-10T14:35:52.349+0000] {standard_task_runner.py:72} INFO - Started process 436 to run task
[2025-03-10T14:35:52.349+0000] {standard_task_runner.py:105} INFO - Job 153: Subtask dbt_run
[2025-03-10T14:35:52.365+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-10T14:35:52.423+0000] {task_command.py:467} INFO - Running <TaskInstance: xkcd_daily.dbt_run manual__2025-03-10T14:20:39.848994+00:00 [running]> on host 157a64d0872f
[2025-03-10T14:35:52.582+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='xkcd_daily' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T14:20:39.848994+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-10T14:20:39.848994+00:00'
[2025-03-10T14:35:52.582+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-10T14:35:52.590+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-10T14:35:52.591+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /dbt/transformations && dbt run --project-dir /dbt/transformations --profiles-dir /dbt']
[2025-03-10T14:35:52.595+0000] {subprocess.py:99} INFO - Output:
[2025-03-10T14:35:53.835+0000] {subprocess.py:106} INFO - [0m14:35:53  Running with dbt=1.9.3
[2025-03-10T14:35:53.997+0000] {subprocess.py:106} INFO - [0m14:35:53  Registered adapter: postgres=1.9.0
[2025-03-10T14:35:54.463+0000] {subprocess.py:106} INFO - [0m14:35:54  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-10T14:35:54.463+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-03-10T14:35:54.464+0000] {subprocess.py:106} INFO - - models.transformations.example
[2025-03-10T14:35:54.558+0000] {subprocess.py:106} INFO - [0m14:35:54  Found 3 models, 1 source, 433 macros
[2025-03-10T14:35:54.560+0000] {subprocess.py:106} INFO - [0m14:35:54
[2025-03-10T14:35:54.560+0000] {subprocess.py:106} INFO - [0m14:35:54  Concurrency: 1 threads (target='prod')
[2025-03-10T14:35:54.560+0000] {subprocess.py:106} INFO - [0m14:35:54
[2025-03-10T14:35:54.653+0000] {subprocess.py:106} INFO - [0m14:35:54  1 of 3 START sql table model dbt_dwh.dim_comics ................................ [RUN]
[2025-03-10T14:35:54.776+0000] {subprocess.py:106} INFO - [0m14:35:54  1 of 3 OK created sql table model dbt_dwh.dim_comics ........................... [[32mSELECT 561[0m in 0.12s]
[2025-03-10T14:35:54.780+0000] {subprocess.py:106} INFO - [0m14:35:54  2 of 3 START sql table model dbt_dwh.fact_cost ................................. [RUN]
[2025-03-10T14:35:54.848+0000] {subprocess.py:106} INFO - [0m14:35:54  2 of 3 OK created sql table model dbt_dwh.fact_cost ............................ [[32mSELECT 561[0m in 0.07s]
[2025-03-10T14:35:54.849+0000] {subprocess.py:106} INFO - [0m14:35:54  3 of 3 START sql table model dbt_dwh.fact_review ............................... [RUN]
[2025-03-10T14:35:54.927+0000] {subprocess.py:106} INFO - [0m14:35:54  3 of 3 OK created sql table model dbt_dwh.fact_review .......................... [[32mSELECT 561[0m in 0.08s]
[2025-03-10T14:35:54.952+0000] {subprocess.py:106} INFO - [0m14:35:54
[2025-03-10T14:35:54.952+0000] {subprocess.py:106} INFO - [0m14:35:54  Finished running 3 table models in 0 hours 0 minutes and 0.39 seconds (0.39s).
[2025-03-10T14:35:54.996+0000] {subprocess.py:106} INFO - [0m14:35:54
[2025-03-10T14:35:54.996+0000] {subprocess.py:106} INFO - [0m14:35:54  [32mCompleted successfully[0m
[2025-03-10T14:35:54.997+0000] {subprocess.py:106} INFO - [0m14:35:54
[2025-03-10T14:35:54.997+0000] {subprocess.py:106} INFO - [0m14:35:54  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2025-03-10T14:35:55.603+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-10T14:35:55.638+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-10T14:35:55.638+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=xkcd_daily, task_id=dbt_run, run_id=manual__2025-03-10T14:20:39.848994+00:00, execution_date=20250310T142039, start_date=20250310T143552, end_date=20250310T143555
[2025-03-10T14:35:55.691+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-10T14:35:55.708+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-10T14:35:55.711+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
